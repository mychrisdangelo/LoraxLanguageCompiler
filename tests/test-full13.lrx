/* 
 * Authors: 
 * Kira Whitehouse
 * Chris D'Angelo
 * End to end test of @ operator without child operator
 */

int main()
{
	tree <int>t(2);
	tree <float>t2(2);
	tree <bool>t3(2);
	tree <char>t4(2);
	t = 1[2, 3];
	t2 = 1.0[2.0, 3.0];
	t3 = true[true, false];
	t4 = 'a'['b', 'c'];
	print(t@, '\n', t2@, '\n', t3@, '\n', t4@);
}