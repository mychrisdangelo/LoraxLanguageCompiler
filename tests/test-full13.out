1
1.000000
true
a